apps/web-giddh/src/app/contact/contact.component.ts

```diff
@@ -302,6 +302,21 @@ export class ContactComponent implements OnInit, OnDestroy {
         }
     }
 
+    /**
+     * This will get output by PO
+     *
+     * @param {boolean} event
+     * @memberof ContactComponent
+     */
+     public getPageHeading(): string {
+        if(this.activeTab === 'aging-report') {
```

Comment:
add if condition here for isMobileScreen. 

example:

if(isMobileScreen) {
    // add all if, else if conditions here
} else {
    return "";
}